templates/bake/Controller/controller.twig

```diff
@@ -18,10 +18,12 @@
  */
 #}
 <?php
+declare(strict_types=1);
 namespace {{ namespace }}\Controller{{ prefix }};
 
+{% if prefix %}
 use {{ namespace }}\Controller\AppController;
-
+{% endif %}
```

Comment:
within a plugin 
with this `if` block , generated controller will use main `AppController` if user does not use `prefix` , and with use of `prefix` controller will use `AppController` of plugin

Reply:
> within a plugin with this if block , generated controller will use main AppController

No, it will use the plugin's `AppController`.